### PR TITLE
feat(config): add new rules

### PR DESCRIPTION
**Pull request by theetrain**
- operator-linebreak and padded-blocks

See #1

### CODE REVIEW

**Comment by darcyclarke** [comment]
I'm okay with these changes but could you stay consistent and use the value `'error'` vs. the `2`. I think it's a bit more clear for anyone that has never looked at a lint file what is configured.

File: index.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ module.exports = {
     'no-alert': 'error',
     'no-console': 'error',
     'no-underscore-dangle': 'off',
+    'operator-linebreak': 2,
```
Comment (darcyclarke) [inline index.js]: I'm okay with these changes but could you stay consistent and use the value 'error' vs. the 2? I think it's a bit more clear for anyone that has never looked at a lint file what is configured.

**Comment by darcyclarke** [comment]
Looking into this a bit. I question how often folks are running into multi-line conditionals. If you can provide a good example of this @theetrain maybe we can discuss here a bit about the value and whether we should be enforcing single line operators (ie. configure with `'none'`).

On the other hand, I've grown to despise `padded-blocks` so I appreciate that change/definition.

**Comment by kretzm** [comment]
padded-blocks: I do like empty lines at the beginning of block statement for clarity, so I am for adding in "always".

operator-linebreak: Like Darcy, not sure I understand the usecase for adding this in.. looking for some examples

**Comment by theetrain** [comment]
Re: operator linebreaks

For string concatenation, I know it's preferred to use template literals. For math, I prefer placing operators at the end of a line. Such as:

```js
const setHeight = (image, mouse) => {
  const newHeight = image.clientHeight +
                    mouse.y +
                    image.scrollTop

  image.style.height = newHeight
}
```

**Comment by darcyclarke** [comment]
@theetrain I don't see why you'd make new lines in this case. Basic math, subjectively, should be able to live on the same line. If you're finding you're hitting the ~standard 80 char line limit then you may be trying to do too much in a single calculation.

```js
const setHeight = (image, mouse) => {
  const newHeight = image.clientHeight + mouse.y + image.scrollTop
  image.style.height = newHeight
}
```
^ reads better to me, personally

**Comment by theetrain** [comment]
I've forgone my recommendation to add `operator-linebreak` and will move forward with just `padded-blocks`.

For what it's worth, TDS team is using Prettier and has so far had positive experiences. In this way, modified code can be properly formatted before running tests.

**Comment by ahmadnassri** [comment]
@theetrain can you bring Prettier to arch forum discussion, if there is value in standardising on one tool, lets maximise the effort!

**Pull request merged**